### PR TITLE
Microsoft Advanced Threat Analytics

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/13975

## Minimum version of Demisto
- [x] 5.0.0

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MicrosoftAdvancedThreatAnalytics/TestPlaybooks/MicrosoftAdvancedThreatAnalytics-Test.yml [hunk 1]
```diff
@@ -0,0 +1,688 @@
+id: Microsoft Advanced Threat Analytics - Test
+version: -1
+vcShouldKeepItemLegacyProdMachine: false
```
[the remark this answers is not in the document]
Comment (Itay4) [reply Packs/MicrosoftAdvancedThreatAnalytics/TestPlaybooks/MicrosoftAdvancedThreatAnalytics-Test.yml:3]: it doesnt really matter

**Comment by Itay4** [comment]
> Update the image to include the product name
> No need for detailed description?
> Didn't go over the code, but i suggest splitting the functions to multiple smaller chunks of code, you have many things going on in the command functions

nothing to mention in the detailed description, just use creds and nothing to set up on the product side

i think splitting the command functions is an overkill, its pretty straight forward so dont see the need here

File: Packs/MicrosoftAdvancedThreatAnalytics/Integrations/MicrosoftAdvancedThreatAnalytics/MicrosoftAdvancedThreatAnalytics.py [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+        return self._http_request(
+            method='DELETE',
+            url_suffix=f'/suspiciousActivities/{suspicious_activity_id}',
+            json_data=body,
+            params=params,
+            ok_codes=(204,),
+            resp_type='text'
+        )
+
+    def get_monitoring_alert_request(self) -> Dict[str, Any]:
```
Comment (yaakovi) [inline Packs/MicrosoftAdvancedThreatAnalytics/Integrations/MicrosoftAdvancedThreatAnalytics/MicrosoftAdvancedThreatAnalytics.py:69]: no why to add the filters to the request?
```suggestion
    def get_monitoring_alerts_request(self) -> Dict[str, Any]:
```

File: Packs/MicrosoftAdvancedThreatAnalytics/Integrations/MicrosoftAdvancedThreatAnalytics/MicrosoftAdvancedThreatAnalytics.py [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+    try:
+        client = Client(
+            base_url=base_url,
+            auth=HttpNtlmAuth(username, password),
+            verify=verify_certificate,
+            proxy=proxy
+        )
+
+        if demisto.command() == 'test-module':
+            result = test_module(client)
```
Comment (yaakovi) [inline Packs/MicrosoftAdvancedThreatAnalytics/Integrations/MicrosoftAdvancedThreatAnalytics/MicrosoftAdvancedThreatAnalytics.py:320]: didn't we agree that the `test-module` should also run the `fetch-incidents` flow when enabled?

**Pull request merged**